### DISCUSSION TITLE
SlingerPlayer Android caches remote webpage [solved]

### DISCUSSION THREAD

**Discussion by bradut** [GerryDazoo/SlingerPlayer#24] [General]
I have noticed that the Android version of SlingerPlayer caches the remote control webpage, and the only way I was able to get it display a different version of that page was to reinstall the app.

That was OK with me and probably with 99.999% of the users.

Now, I have developed a RC webpage that connects to different WebAPIs 
- to get updated TV programs info,
- to send channel change commands to the Slingbox Server,
- to get the streaming status of a Slingbox via SignalR in order to display its selected channel.

This RC webpage works fine when launched from the Windows version of SlingerPlayer or directly in a browser

Unfortunately, in SlingerPlayer for Android it often cannot connect to the servers.

I believe these issues may be a consequence of cache-ing but not sure 

So, I am wondering: is it possible to disable cache?

**Comment by GerryDazoo** [comment]
https://cristian.sulea.net/blog/disable-browser-caching-with-meta-html-tags/

 

 

From: Bradut Dima ***@***.***> 
Sent: Monday, September 11, 2023 2:10 PM
To: GerryDazoo/SlingerPlayer ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GerryDazoo/SlingerPlayer] SlingerPlayer Android caches remote webpage (Discussion #24)

 

I have noticed that the Android version of SlingerPlayer caches the remote control webpage, and the only way I was able to get it display a different version of that page was to reinstall the app.

That was OK with me and probably with 99.999% of the users.

Now, I have developed a RC webpage that connects to different WebAPIs

*	to get updated TV programs info,
*	to send channel change commands to the Slingbox Server,
*	to get the streaming status of a Slingbox via SignalR in order to display its selected channel.

This RC webpage works fine when launched from the Windows version of SlingerPlayer or directly in a browser

Unfortunately, in SlingerPlayer for Android it often cannot connect to the servers.

I believe these issues may be a consequence of catching, but not sure

So, I am wondering: is it possible to disable catching?

**Reply by bradut** [reply]
Thank you Gerry @GerryDazoo , the fix indicated by you works  !

**Reply by bradut** [reply]
**Update**: 
To effectively clean the cache, it is required to **clear the SlingerPlayer app's cache** on the phone/tablet. 
I used to clear all the data, which deleted the URLs, but recently, I noticed that cleaning the cache was sufficient.
